pub-matic-htb.js

```diff
@@ -68,20 +68,66 @@ function PubMaticHtb(configs) {
     var __profile;
     var __globalConfigs;
 
+    const PUBMATIC_DIGITRUST_KEY = 'nFIn8aLzbd';
+
     /* =====================================
      * Functions
      * ---------------------------------- */
 
     /* Utilities
      * ---------------------------------- */
-     function __populateImprObject(returnParcels) {
+
+    function __getDigiTrustObject(key){
+        function getDigiTrustId(key) {
+            let digiTrustUser;
+            if(Browser.topWindow.DigiTrust) {
+                digiTrustUser=Browser.topWindow.DigiTrust.getUser({
+                    member: key
+                });
+            }
+            return (digiTrustUser && digiTrustUser.success && digiTrustUser.identity) ? digiTrustUser.identity : null;
+        }
+        let digiTrustId = getDigiTrustId(key);
+        // Verify there is an ID and this user has not opted out
+        if (!digiTrustId || (digiTrustId.privacy && digiTrustId.privacy.optout)) {
+            return null;
+        }
+        return digiTrustId;
+    }
+
+    function __handleDigitrustId(eids) {
+        let digiTrustId = __getDigiTrustObject(PUBMATIC_DIGITRUST_KEY);
+        if (digiTrustId && digiTrustId !== null) {
+            eids.push({
+                'source': 'digitru.st',
```

Comment:
can we move this to a constant var?

Reply:
you mean to change let to var or store this value in const ?